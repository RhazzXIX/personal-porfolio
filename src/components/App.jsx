import React from "react"

export default function App () {
  return(
   <>
    <h1>Hello Up-skilling World!</h1>
   </>
  )
}