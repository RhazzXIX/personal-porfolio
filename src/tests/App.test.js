import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import App from "../components/App";
import React from "react";

describe('App component', () => {
  it('Renders on screen', () => {
    const {container} = render(<App />)
    expect(container).toMatchSnapshot()
  })
})