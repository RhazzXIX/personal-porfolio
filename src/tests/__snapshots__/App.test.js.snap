// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Renders on screen 1`] = `
<div>
  <h1>
    Hello Up-skilling World!
  </h1>
</div>
`;
